### PR TITLE
ARROW-15074: [Format] Clarify that LZ4 contains a single frame

### PR DESCRIPTION
**Pull request by jorgecarleitao**
See associated discussion at https://issues.apache.org/jira/browse/ARROW-15074

### CODE REVIEW

File: format/Message.fbs [hunk 1]
```diff
@@ -70,7 +70,7 @@ enum BodyCompressionMethod:byte {
 /// bodies. Intended for use with RecordBatch but could be used for other
 /// message types
 table BodyCompression {
-  /// Compressor library
+  /// Compressor library. Note that for LZ4_FRAME, it must contain a single frame
```
Comment (pitrou) [inline format/Message.fbs:73]:
```suggestion
  /// Compressor library.
  /// For LZ4_FRAME, each compressed buffer must consist of a single frame.
```

**Review by pitrou — COMMENTED** [review]
Thanks @jorgecarleitao ! Just a suggestion below, feel free to improve it :-)

File: format/Message.fbs (hunk 1 above)
Comment (jorgecarleitao) [reply format/Message.fbs]: Good one. Committed. Thanks!

**Comment by emkornfield** [comment]
Why is this the case?  Is this just adapting what implementations currently do?

**Comment by pitrou** [comment]
At least the C++ implementation, yes. Basically, each LZ4 frame is meant to be an independent stream, and you can streaming compression/decompression even with a single frame.

**Pull request closed without merging**